browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -1565,6 +1584,38 @@ TEST_F(KeyringServiceUnitTest, CreateAndRestoreWallet) {
   EXPECT_FALSE(observer.IsKeyringRestored(mojom::kSolanaKeyringId));
 }
 
+TEST_F(KeyringServiceUnitTest, DefaultSolanaAccountCreated) {
+  KeyringService service(json_rpc_service(), GetPrefs());
```

Comment:
Why does this not need `create-default-solana-account` to be true?

Reply:
It is true by default

Reply:
Removed this